data/modelCuration/transRxnNewGPR/TransRxnNewGPR.tsv

```diff
@@ -0,0 +1,29 @@
+rxnID	new_gpr	SGD	uniport	yeastcyc	kegg	TCDB	argument
+r_1108	(YER056C or YER060W or YER060W-A)	https://www.yeastgenome.org/locus/S000003154	https://www.uniprot.org/uniprot/P17064; https://www.uniprot.org/uniprot/P53099 https://www.uniprot.org/uniprot/Q12119 https://www.uniprot.org/uniprot/P40039	https://biocyc.org/gene?orgid=YEAST&id=G3O-30671	https://www.genome.jp/entry/sce:YGL186C		YGL186C is Vitamin transport not adenine
+r_1134	(YDL245C or YDR342C or YDR343C or YDR345C or YEL069C or YFL011W  or YHR096C or YJL214W or YJL219W or YJR158W  or YNR072W or YOL156W)	https://www.yeastgenome.org/locus/S000001134; https://www.yeastgenome.org/locus/S000005516 https://www.yeastgenome.org/locus/S000001136	https://www.uniprot.org/uniprot/P32467; https://www.uniprot.org/uniprot/P32465�� https://www.uniprot.org/uniprot/P54862	https://biocyc.org/gene?orgid=YEAST&id=G3O-31139	https://www.genome.jp/entry/sce:YHR092C; https://www.genome.jp/entry/sce:YHR094C; https://www.genome.jp/entry/sce:YOL156W		YHR092C, YHR094C, YOL156W could transport glucose, but not fructose
```

Comment:
Double check this one. According to the SGD GO annotation, these three genes have functions for fructose transport, especially for the gene [YOL156W](https://www.yeastgenome.org/locus/S000005516)